src/libraries/Common/src/System/Text/Json/PooledByteBufferWriter.cs

```diff
@@ -219,5 +224,12 @@ public static void ThrowOutOfMemoryException_BufferMaximumSizeExceeded(uint capa
         {
```

Comment:
Kind of strange these don't live in `ThrowHelper.cs`, but I guess it predates this change.